agent/engine/interface.go

```diff
@@ -32,13 +32,16 @@ type TaskEngine interface {
 	// TaskEvents will provide information about tasks that have been previously
 	// executed. Specifically, it will provide information when they reach
 	// running or stopped, as well as providing portbinding and other metadata
-	TaskEvents() (<-chan api.TaskStateChange, <-chan api.ContainerStateChange)
+	TaskEvents() (chan api.TaskStateChange, chan api.ContainerStateChange)
```

Comment:
This is because in the `eniwatcher` we need to reuse this channel to write eniattachment event to eventhandler. Since this has changed we will get rid of this after the implementation of single channel provided by the `eventhandler`.